### DISCUSSION TITLE
How to change service key

### DISCUSSION THREAD

**Discussion by nezzard** [supabase/supabase#12525] [Questions]
Hi generated anon and service key https://supabase.com/docs/guides/self-hosting#api-keys
than I replace it in .env file, than rebuild and restart containers, but anyway I can access REST API with old default keys

**Comment by GaryAustin1** [comment]
Are you saying you can still access with your old keys or are you saying you want to access with your old keys?

If you don't change the jwt secret then the keys should be exactly the same.  If you did change the secret then you need to update that also in your .env file.
